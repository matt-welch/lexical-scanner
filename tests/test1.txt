
111    +-+*
111...111*****+++++ 111.abc
111.110 011

111

12.3e+10
45.2E10