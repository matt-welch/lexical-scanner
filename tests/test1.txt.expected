2 111 
2 +
2 -
2 +
2 *
3 111 
3 .
3 .
3 .
3 111 
3 *
3 *
3 *
3 *
3 *
3 +
3 +
3 +
3 +
3 +
3 111 
3 .
3 abc 
4 111.110 
4 0 
4 11 
6 111 
8 12.3e+10 
9 45.2 
9 E10 
